test001.c: 函数 `f' :
test001.c:6: エラー: `a' は宣言されていません.
test001.c:7: エラー: `b' は宣言されていません.
test001.c:8: エラー: `c' は宣言されていません.
test001.c:8: エラー: `d' は宣言されていません.
test001.c: 函数 `g' :
test001.c:13: エラー: `a' は宣言されていません.
test001.c:13: エラー: `b' は宣言されていません.
test001.c:13: エラー: `c' は宣言されていません.
test001.c:13: エラー: `d' は宣言されていません.
